### PR TITLE
[doc] Add contributing rules

### PR DESCRIPTION
**Pull request by NikitaZotov**
* [x] Read PR [documentation](https://github.com/ostis-ai/sc-machine/blob/main/docs/dev/pr.md/)
* [x] Update changelog
* [x] Update documentation

### CODE REVIEW

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]
Looks like file should be refactored. Commit examples, Pull Request section, etc.
image

File: docs/index.md [hunk 1]
```diff
@@ -21,6 +21,6 @@
     * [Knowledge Base builder](sc-tools/kb_builder.md)
 
 - **Development**
-    * [Pull Request](dev/pr.md)
+    * [Pull Request](../CONTRIBUTING.md)
```
Comment (KovalM) [inline docs/index.md:24]: We need to delete dev/pr for support these rules in single place

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+This page describes rules to contribute changes and features by Pull Requests creating.
+
+### Commits message format
+
+Each commit message should be formed as: [tag1]...[tagN] Message text (#issue)
+
+Message text should start from an upper case letter. If commit doesn't fix or implement any #issue, then it shouldn't be pointed in commit message Examples:
+
+[cpp] Colored log output
```
Comment (KovalM) [inline CONTRIBUTING.md:9]: Revert formatting from dev/pr.md

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+### Pull Request Review
+
+ - Reviewer should test code from PR if CI don't do it
+ - Reviewer submit review as set of conversations
+ - Author make review fixes at `Review fixes` commits
+ - Author re-request review
+ - Reviewer resolve conversations if they were fixed and approve PR
+
+### After Pull Request Review
```
Comment (KovalM) [inline CONTRIBUTING.md:87]: Not all contributes can do it

**Review by KovalM — CHANGES_REQUESTED** [review]

**Review by AlexandrZagorskiy — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (KovalM) [reply CONTRIBUTING.md:95]: Not fixed

**Review by KovalM — CHANGES_REQUESTED** [review]

**Pull request merged**